spec/traits.dd

```diff
@@ -1713,6 +1714,39 @@ $(H2 $(GNAME toType))
 
         $(RATIONALE Provides the inverse operation of the $(DDSUBLINK property, mangleof, `.mangleof`) property.)
 
+$(H2 $(GNAME initSymbol))
+
+        $(P Takes a single argument, which must evaluate to a `class` or `struct` type.
+            Returns a `const(void)[]` that points to either the initializer symbol of the supplied
+            type or `null` if it is zero-initialized. The referenced symbol holds the initial state
+            of any instance of the supplied type.
```

Comment:
Can we refer to something about the length of the returned slice? From the example below, I understood that `__traits(initSymbol, C)` returns the correct length of the initializer memory even if zero-initialized and that matches the implementation, although this wording can be interpreted wrongly, IMO.

Reply:
Added a more explicit description for the `ptr` and `length` properties of  the returned slice.